### DISCUSSION TITLE
Feature Request:  under-voltage

### DISCUSSION THREAD

**Discussion by cniesen** [Pi4J/pi4j#410] [Ideas]
The `vcgencmd get_throttled` command line command provides information if the Pi currently experiences under voltage or has experienced it in the past.  It also provides some info if the board is throttled.

Interpreting the hex result is well documented in https://gist.github.com/Paraphraser/17fb6320d0e896c6446fb886e1207c7e#the-vcgencmd_power_reportsh-script  I think a similar code could be included in this library.

I think the `BoardReading` class would be a good place to add the raw and the interpreted (human friendly) under-voltage status of the board.

**Comment by eitch** [comment]
This is brilliant, thanks. We'll look into it. Unless you want to write up a PR for adding it to the BoardReading class?

**Comment by dariuszzbyrad** [comment]
[PR with changes](https://github.com/Pi4J/pi4j-v2/pull/432)
